core/node/node_framework/src/implementations/layers/house_keeper.rs

```diff
@@ -59,8 +70,26 @@ impl WiringLayer for HouseKeeperLayer {
             replica_pool.clone(),
         );
 
+        let app_health = input.app_health.0;
+        app_health
+            .insert_custom_component(Arc::new(NodeVersionInfo::default()))
```

Comment:
I think we have to include it into every component, not only to house keeper

Reply:
I.e., include it in a separate layer, right? (I'd suggest to extend `PostgresMetricsLayer`, renaming it to `PostgresLayer`.)

Reply:
Moved database health task to PostgresLayer in https://github.com/matter-labs/zksync-era/pull/3193/commits/5e235b13c54baac79e8dbc05b0bdd59c1650b407